src/Query.php

```diff
@@ -203,25 +203,31 @@ public function buildCursor($db = null)
 
     /**
      * Fetches rows from the given Mongo cursor.
-     * @param \MongoDB\Driver\Cursor $cursor Mongo cursor instance to fetch data from.
+     * @param yii\mongodb\Connection $db the MongoDB connection used to fetch rows.
      * @param bool $all whether to fetch all rows or only first one.
      * @param string|callable $indexBy the column name or PHP callback,
      * by which the query results should be indexed by.
      * @throws Exception on failure.
      * @return array|bool result.
      */
-    protected function fetchRows($cursor, $all = true, $indexBy = null)
+    protected function fetchRows($db, $all = true, $indexBy = null)
     {
+        $db = $db === null ? yii::$app->mongodb : $db;
```

Comment:
What is the purpose of this change?

Reply:
like first your question.
default db connection is null and default value for null is always `yii::$app->mongodb`.